src/App.js

```diff
@@ -1,13 +1,15 @@
+import React, { Component } from 'react';
+import DoCalculation from './components/calculator';
 import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Math Magician</h1>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
Always make it a habit to fix linter errors. Please fix that error.

Reply:
Hello afanimartin, I have used that line because in first project requirement was not to use function